### DISCUSSION TITLE
CSS Inline Styling

### DISCUSSION THREAD

**Discussion by donbwhite** [GrapesJS/grapesjs#4228] [Q&A]
I'm working with GrapesJS for an email builder and I'm having trouble with figuring out how to enable inline CSS styling as opposed to the classes being created. I checked the repo already and did see some good points made by @artf so I have the grapesjs-preset-newsletter plugin already and I see there's the gjs-get-inlined-html function in order to get the html with the CSS inline, but where I'm getting stuck is how to take the resulting CSS and make that the html that gets saved by the store function.

I'm kind of stuck as to what the proper method would be. Any ideas?

**Comment by artf** [comment]
I guess this should work
```js
editor.on('storage:start:store', data => {
    data.inlinedHTML = editor.runCommand('gjs-get-inlined-html'); 
});
```
